development.md

```diff
@@ -1,152 +1,109 @@
-# Improve development guide 
+# Development Setup
 
-# Development setup
+This is a guide to running the Zulip desktop app from source,
+in order to contribute to developing it.
 
-This is a guide to running the Zulip desktop app from a source tree, in order to contribute to developing it. The Zulip electron development environment can be installed on **macOS, Windows, and Linux** (Debian or Ubuntu recommended). You’ll need at least **2GB of available RAM**. Installing the Zulip electron development environment requires downloading several hundred megabytes of dependencies, so you will need an **active, reasonably fast, internet connection throughout the entire installation processes.**
+## Prerequisites
 
-# Set up Git & GitHub
+To build and run the app from source, you'll need the following:
 
-You can skip this step if you already have Git, GitHub.
+* [Git](http://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+  * Use our [Git Guide](https://zulip.readthedocs.io/en/latest/git/setup.html) to get started with Git and GitHub.
+* [Node.js](https://nodejs.org) >= v6.9.0
+  * [NPM](https://www.npmjs.com/get-npm) and
+    [node-gyp](https://github.com/nodejs/node-gyp#installation),
+    if they don't come bundled with your Node.js installation
+* [Python](https://www.python.org/downloads/release/python-2713/)
+  (v2.7.x recommended)
+* A C++ compiler compatible with C++11
+* Linux users also need [Snapcraft](https://snapcraft.io/)
+* Development headers for the libXext, libXtst, and libxkbfile libraries
```

Comment:
Can include the command for it somewhere, at least in Linux `$ sudo apt install libxext-dev libxtst-dev libxkbfile-dev libgconf-2-4`.

Reply:
Do we also need to add `$ sudo apt install build-essential`?